install/threesdk/lib/SDKContainers.py

```diff
@@ -6,20 +14,78 @@ def __init__(self, core, args):
         self.core = core
         self.args = args
 
-    def _identity_ask(self, identity=None):
-        if not identity and self.args.identity:
-            return self.args.identity
-        if not identity:
-            identity = self.core.IT.Tools.ask_string("what is your threebot name (identity)?")
-        if "." not in identity:
-            identity += ".3bot"
-        identity = identity.lower()
-        if self.args.identity != identity:
-            self.args.identity = identity
-            self.args.words = None
-            self.args.email = None
-
-        return identity
+    def _get_user(self):
+        resp = requests.get("https://{}/explorer/users".format(self.args.explorer), params={"name": self.args.identity})
+        if resp.status_code == 404:
+            return None
+        else:
+            users = resp.json()
+            if users:
+                return users[0]
+            return None
+
+    def _identity_ask(self, identity=None, explorer=None):
+        def _fill_identity_args(identity, explorer):
+            def fill_words():
+                words = self.core.IT.Tools.ask_string("what are the words associated with your identity?")
```

Comment:
shouldn't we more explicit and says, copy the words from your 3bot connect app ?
Also for the threebot name, it should be the same as in the 3bot app.

Reply:
Let's make this:

"Copy the phrase from your 3bot Connect app here"